playbooks/mksysb.yml

```diff
@@ -0,0 +1,63 @@
+---
+- name: "Run Mksysb"
+  hosts: aixhosts
+  become: yes
+  become_user: root
+  vars:
+   mksysb_lv: "backuplv"
+   mksysb_dir: "/backup"
+   NIM_Server: "nimhost"
+   backup_repo: "/mksysb_images"
+   NFS_mount: "/mnt"
+  tasks:
+   - name: "check backup filesystem "
+     aix_filesystem:
+      device: "{{ mksysb_lv }}"
+      filesystem: "{{ mksysb_dir }}"
+      size: 10G
+      vg: datavg
+      state: present
+     when: ansible_distribution == 'AIX'
+   - name: "Mount backup filesystem"
+     aix_filesystem:
+      device: "{{ mksysb_lv }}"
+      filesystem: "{{ mksysb_dir }}"
+      state: mounted
+   - name: "Run mksysb backup"
+     mksysb:
+      name: "{{ansible_hostname}}_mksysb_{{ansible_facts['date_time']['date']}}"

```

Comment:
I would prefer "{{ ansible_date_time.date }}" instead of "{{ansible_facts['date_time']['date']}}"

Reply:
Updated..!!! and it looks good thanks for the suggestion..